### DISCUSSION TITLE
Training BRISQUE model on custom dataset

### DISCUSSION THREAD

**Discussion by spaul13** [rehanguha/brisque#7] [Q&A] [answered]
can anyone please help me telling how can I train BRISQUE model on my custom dataset?

**Comment by rehanguha** [comment]
Hi @spaul13,
Did not understand by the question which you have asked as there is nothing to train in BRISQUE. BRISQUE is an Image Quality Evaluator.
You can take a look at https://www.mathworks.com/help/images/ref/brisque.html

**Comment by spaul13** [comment]
@rehanguha, in BRISQUE there is a SVM model which is trained on TID2008 dataset in original paper. Please refer to this link https://www.mathworks.com/help/images/train-and-use-a-no-reference-quality-assessment-model.html 

I am asking for how can I train the BRISQUE SVM model on my dataset.

**Comment by rehanguha** [comment]
Oh. Got it now @spaul13 . Sorry, Didn't get your question at first.
Yup, if you look at the code.
`model = svmutil.svm_load_model('brisque_svm.txt')`

I did not get time to integrate LibSVM. But using LibSVM you can train you SVM model.

https://github.com/cjlin1/libsvm/tree/master/python

I will start working to integrate this part to this project.

**Comment by rehanguha** [comment, marked as answer]
v0.2.0 has the required feature.
